### PR TITLE
Adding Rationales page

### PR DESCRIPTION
**Pull request by gene1wood**

### CODE REVIEW

File: Security/Guidelines/Rationales.mediawiki [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+</td>
+</tr></table>
+
+= Rationales =
+{| class="wikitable"
+!Topic
+!Rationale
+|-
+|<div id="shared-passwords">[[#shared-passwords|§]] Shared passwords</div>
+|Shared passwords are passwords that more than one person knows or has access to. They're discouraged beacause
```
Comment (gdestuynder) [inline Security/Guidelines/Rationales.mediawiki]: typo because

File: Security/Guidelines/Rationales.mediawiki [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+* Use of them makes auditing access difficult because
+** multiple users appear in audit logs as one user and different users actions are difficult to differentiate
+** OpSec must look through all logs that all users who use the shared password caused entries in
```
Comment (gdestuynder) [inline Security/Guidelines/Rationales.mediawiki]: would reword, had to read it 3x to understand.

File: Security/Guidelines/Rationales.mediawiki [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+** correlation of events across different systems is impossible if multiple people are creating event records with a single shared account across multiple systems
```
Comment (gdestuynder) [inline Security/Guidelines/Rationales.mediawiki]: "at the same time" since we can sometimes differenciate if we know who used the login at a certain time, and if no one else has logged in

File: Security/Guidelines/Rationales.mediawiki [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+* Revoking access to a subset of the users of a shared password requires a password change that affects all usrs
```
Comment (gdestuynder) [inline Security/Guidelines/Rationales.mediawiki]: typo users

**Comment by gdestuynder** [comment]
I wonder if this should instead be called "definitions"
Let's bring this up at the next opsec weekly for review/etc.

**Comment by gene1wood** [comment]
@gdestuynder Here's the name we agreed on and it's new location. If this looks good feel free to merge.
